docs/ref/models/expressions.txt

```diff
@@ -371,7 +372,8 @@ SQL that is generated. Here's a brief example::
                 expression,
                 distinct='DISTINCT ' if distinct else '',
                 output_field=IntegerField(),
-                **extra)
+                **extra,
```

Comment:
That might not be a syntax error but it does suggest that one could add additional items after `**extra`, which would be a syntax error.